packages/story-editor/src/app/canvas/useCanvasKeys.ts

```diff
@@ -233,7 +238,11 @@ function useCanvasKeys(ref) {
       }
 
       const { type, id, isLocked } = selectedElements[0];
-      const { hasEditMode, hasEditModeIfLocked } = getDefinitionForType(type);
+      const elDefinition = getDefinitionForType(type);
+      if (!elDefinition) {
```

Comment:
Hmm, annoying, but it's defined as only partially covering the types, though we know it isn't, so it can't be any different, I guess.

Maybe we could change `getDefinitionForType` to throw if not found? It should never happen, and would make using it a lot easier.

Reply:
Something like this should work:

```ts
function getDefinitionForType(type: ElementType) {
  const definition = elementTypes[type];
  if (!definition) {
    throw new Error(`No element definition found for '${type}'`);
  }
  return definition;
}

Reply:
(in fact, most of our code would break if that happened anyway, so we're not introducing new breaks, I think)